src/Microsoft.Diagnostics.DebugServices.Implementation/ModuleService.cs

```diff
@@ -238,7 +238,7 @@ internal PEImage GetPEInfo(ulong address, ulong size, ref PdbInfo pdbInfo, ref V
         /// <param name="version">the PE version or null</param>
         /// <param name="flags">module flags</param>
         /// <returns>PEImage instance or null</returns>
-        private PEImage GetPEInfo(bool isVirtual, ulong address, ulong size, ref PdbInfo pdbInfo, ref VersionInfo? version, ref Module.Flags flags)
+        private PEImage GetPEInfo(bool isVirtual, ulong address, ulong size, ref PdbInfo pdbInfo, ref VersionInfo version, ref Module.Flags flags)
```

Comment:
hmm do we want to pass in a version and only modify if it's not null? I am not sure if this would be better off as an out param and have the caller resolve it.

Reply:
Yes, there are times that version is already set on the way in and GetPEInfo doesn't need to parse and get it.